app/src/main/java/com/github/siela1915/bootcamp/FragmentIngredientCheck.java

```diff
@@ -0,0 +1,124 @@
+package com.github.siela1915.bootcamp;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.text.Selection;
+import android.text.Spannable;
+import android.text.SpannableString;
+import android.text.SpannableStringBuilder;
+import android.text.Spanned;
+import android.text.method.LinkMovementMethod;
+import android.text.style.ClickableSpan;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.github.siela1915.bootcamp.Recipes.ExampleRecipes;
+import com.github.siela1915.bootcamp.Recipes.Ingredient;
+import com.github.siela1915.bootcamp.Recipes.Recipe;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link FragmentIngredientCheck#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class FragmentIngredientCheck extends Fragment {
+
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    public static final String ARG_INGREDIENT_LIST = "ingredients";
+    //private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    //private String mParam1;
+    //private String mParam2;
+    //private Recipe recipe;
```

Comment:
These commented out lines might be removed if they are of no use to keep the codebase clean.